### PR TITLE
task_DEV-1

### PR DESCRIPTION
**Pull request by VadimBelkin**

### CODE REVIEW

File: OutputofArray.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+public class OutputofArray {
```
Comment (AlexanderSoroka) [inline OutputofArray.java]: codestyle violation (naming convention)

File: OutputofArray.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+public class OutputofArray {
+    public static void main(String[] args) {
+        int i;
```
Comment (AlexanderSoroka) [inline OutputofArray.java]: codestyle violation (4.2)

File: OutputofArray.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+public class OutputofArray {
+    public static void main(String[] args) {
+        int i;
+        int []a;
```
Comment (AlexanderSoroka) [inline OutputofArray.java]: array is useless here

File: OutputofArray.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    public static void main(String[] args) {
+        int i;
+        int []a;
+        a = new int[101];
+        for(i = 0; i <= 100; i++) {
+            a[i] = i;
+            if (a[i] % 3 == 0) {
+                a[i] = a[i] / 3;
+                System.out.println("3 * " + a[i]);
+           }
```
Comment (AlexanderSoroka) [inline OutputofArray.java]: codestyle violation (4.1)

File: OutputArray.java [hunk 5]
```diff
@@ -0,0 +1,11 @@
+public class OutputArray {
```
Comment (AlexanderSoroka) [inline OutputArray.java]: codestyle violation (4.2)

Comment (VadimBelkin) [inline OutputArray.java]: I dont understand the problem. I have to spaces, like in (4.5.2).
And I don't understund the difference between (4.2) and (4.5.2).

Comment (AlexanderSoroka) [inline OutputArray.java]: You use 4 spaces to indent blocks. 4.5.2 is about line continuation - feel the difference continue line vs new line

**Pull request merged**